headers/functions/overlay29.h

```diff
@@ -252,5 +252,6 @@ void LogMessageWithPopup(struct entity* user, char* message);
 void LogMessage(struct entity* user, char* message, bool show_popup);
 void LogMessageById(struct entity* user, int message_id, bool show_popup);
 bool RunDungeonMode(undefined4* param_1, undefined4 param_2);
+void SetBothScreensWindowColorToDefault();
```

Comment:
Add `void` to the param list, i.e., `void SetBothScreensWindowColorToDefault(void);`

Unlike in C++, an empty param list in C means unspecified rather than explicitly no params.

Reply:
Huh, that's interesting actually. Apparently it's something done to allow cross compilation with older programs? I had no idea!